server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/PushDataHandler.scala

```diff
@@ -127,6 +130,7 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     } else {
       workerSource.startTimer(WorkerSource.SlavePushDataTime, key)
     }
+    rpcSource.updateMessageMetrics(pushData, pushData.body().size())
```

Comment:
I think we'd better call this method inside receive to be more centralized

Reply:
> I think we'd better call this method inside receive to be more centralized

Done